Resources/Prototypes/Entities/Constructible/Furniture/seats.yml

```diff
@@ -119,7 +119,7 @@
     shapes:
       - !type:PhysShapeAabb
         bounds: "-0.45, -0.3, 0.35, 0.3"
-        layer: [ MobMask ]
+        layer: [ Passable ]
```

Comment:
I pinged paul on this on discord as this really shouldn't be a layer IMO.

Reply:
its correct, that layer is needed for the singulo. at least at the time of the pr, unsure if now